quarkchain/cluster/root_state.py

```diff
@@ -404,13 +415,8 @@ def is_same_chain(self, longer_block_header, shorter_block_header):
 
     def validate_block(self, block):
         """Raise on validation errors """
-        if block.header.version != 0:
-            raise ValueError("incorrect root block version")
-
-        if not self.db.contain_root_block_by_hash(block.header.hash_prev_block):
```

Comment:
why remove this check?

Reply:
Because all the check is now in validate_block_header()